### PR TITLE
Feature/ai 85/fix confidence chain

### PR DESCRIPTION
**Pull request by simdage**

### CODE REVIEW

**Comment by simdage** [comment]
- Couldn't verify the fix of the confidence chain due to UI bug (I'll leave the ticket opened)
- Tackled AI-81 at the same time, removing insurance plan filter which isn't being used.

File: croptalk/prompts_llm.py [hunk 1]
```diff
@@ -28,7 +28,8 @@
 
 REPHRASE_TEMPLATE = """\
 Given the following conversation and a follow up question, identify whether the question is a follow-up question.\
-If it is, rephrase the question to be a standalone question. If it is not, return the question as is.\
+If it is, rephrase the question to be a standalone question. If it is not, return the question as is and nothing else.\
```
Comment (dtourillon-cropguard) [inline croptalk/prompts_llm.py:31]: So this is the tentative fix for the "fix confidence chain" issue?

File: _scripts/evaluate_doc_retrieval.py [hunk 2]
```diff
@@ -78,7 +78,6 @@ def run_use_case(model: Runnable, eval_use_case: NamedTuple, output_df: pd.DataF
         with tracing_v2_enabled() as langchain_tracer:
             model.invoke({
                 "chat_history": [],
-                "rendered_tools" : [],
```
Comment (dtourillon-cropguard) [inline _scripts/evaluate_doc_retrieval.py:81]: Is this part of the tentative fix for the "fix confidence chain" issue too?

Comment (simdage) [reply _scripts/evaluate_doc_retrieval.py:81]: No, this was to be removed from before. It was breaking the code. I added this line in the first tool PR.

File: croptalk/prompts_llm.py (hunk 1 above)
Comment (simdage) [reply croptalk/prompts_llm.py:31]: Yes, do you want me to include it in another PR?

**Review by dtourillon-cropguard — APPROVED** [review]
LGTM, provided it's now tested and works

**Pull request merged**